bundles/org.openhab.binding.telegram/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="telegram"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="telegramBot">
+		<label>Telegram Bot</label>
+		<description>Thing to receive the latest message send to a Telegram Bot.</description>
+
+		<channels>
+			<channel id="lastMessageText" typeId="lastMessageText" />
+			<channel id="lastMessageDate" typeId="lastMessageDate" />
+			<channel id="lastMessageName" typeId="lastMessageName" />
+			<channel id="lastMessageUsername" typeId="lastMessageUsername" />
+			<channel id="chatId" typeId="chatId" />
+			<channel id="replyId" typeId="replyId" />
+		</channels>
+
+		<config-description>
+			<parameter name="botToken" type="text" required="true">
+				<label>botToken</label>
+				<description>Enter the BotToken you received from the "BotFather".</description>
+			</parameter>
+			<parameter name="chatIds" type="text" required="true" multiple="true">
+				<label>chatIds</label>
+				<description>Enter your ChatID. Only Messages from this ID will be send to OpenHAB.</description>
+			</parameter>
+			<parameter name="parseMode" type="text" required="false">
+				<label>parseMode</label>
+				<description>Support for formatted messages, values: Markdown or HTML. Default: no formatting is used.</description>

```

Comment:
Make this an options parameter? Something like this:
```suggestion
				<description>Support for formatted messages, values: Markdown or HTML. Default: no formatting is used.</description>
				<options>
					<option value="">No Formatting</option>
					<option value="HTML">HTML</option>
					<option value="Markdown">Markdown</option>
				</options>
```

Reply:
Thanks, good improvement! I changed it now.